code/frontend/src/components/header.css

```
@@ -1,45 +1,59 @@
-:root {
-  --Rotkehlchen-gray: #e7ecf2;
-  --Rotkehlchen-brown1: #a28d7a;
-  --Rotkehlchen-orange-default: #e79a0e;
-}
-
 .base-header {
-  width: 100vw;
-  display: flex;
-  height: 50px;
-  justify-content: space-between;
-  align-items: center;
-  flex-shrink: 0;
-  border-bottom: 3px solid var(--Rotkehlchen-brown-light);
-  background: var(--Rotkehlchen-gray);
-  position: fixed;
-  top: 0;
-  left: 0;
-  z-index: 1000;
+	z-index: 10;
+	width: 100vw;
+	display: flex;
+	height: var(--header-height);
+	justify-content: space-between;
+	align-items: center;
+	flex-shrink: 0;
+	position: fixed;
```

```position: fixed:``` kannst du zu `position: sticky;` machen dann geht der content am Anfang auch nicht unter den Header.
```suggestion
	position: sticky;
```